websites/B/Bungie/presence.ts

```diff
@@ -0,0 +1,303 @@
+
+var presence = new Presence({
+  clientId: "711393222252822539"
+});
+var browsingStamp = Math.floor(Date.now() / 1000);
+var title;
+var titleTwo;
+var titleThree;
+var titleFour;
+var titleFive;
+var titleSix;
+var titleSeven;
+var titleEight;
+var titleNine;
+var titleTen;
+var titleEleven;
+var titleTwelve;
+var titleThirteen;
+var titleFourteen;
+var titleFifteen;
+var titleSixteen;
+var titleSeventeen;
+var titleEighteen;
+var titleNinteen;
+var titleTwenty
+var titleTwentyOne;
+var titleTwentyTwo;
+var titleTwentyThree;
+var titleTwentyFour
+presence.on("UpdateData", async () => { // You had it like {} and forgot to put the code used in those two brackets
+  const presenceData: presenceData = {
+    largeImageKey: "logo"
+  };
+console.log ("Test")
+  //This one is for posts that are made in the /news section of Bungie.net
+  //This one is for the title of forum posts on Bungie.net
+  //This one is for posts that are on the featured section of the homepage on Bungie.net. 
+  //let titleThree = document.querySelector("#article-container > h1")
+  //All of these statements will be the menu buttons first. Just to make it easier on myself. 
+  let path = document.location.pathname;
+  //titleFive = document.querySelector("#clanSideBar > div.container-left.customScroll.customScrollOff > a > div.compact-clanidentity-containter > div.clanNameContainer > h2 > span")
+  if (window.location.hostname == "bungie.net" || window.location.hostname == "www.bungie.net"){
+    presenceData.startTimestamp = browsingStamp;
+    presenceData.details = ("Viewing the Bungie.net Homepage")
+  }
+  else if (window.location.hostname == "comics.bungie.net"){
+    presenceData.startTimestamp = browsingStamp;
+    presenceData.details = ("Readig Destiny Comics")
+  }
+    //else if (path.includes("/en/News/") && title) {
+  //presenceData.startTimestamp = browsingStamp;
+  //presenceData.details = ("Looking at " + title.textContent);
+  //}
```

Comment:
why this not enabled?

Reply:
That was for a different thing that would break this presence...